Bottom: 2ab79436b858f0bf24ff91555665a0823faef381
Top:    f7098734b90e3bb9a92221c046446891d12d7264
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-08-07 17:27:58 -0700

Refresh of time-ex

---

diff --git a/Makefile.am b/Makefile.am
index e1be9f3..08582d3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -54,6 +54,8 @@ examples_rcopy_SOURCES = examples/rcopy.c
 examples_rcopy_LDADD = $(top_builddir)/src/librdmacm.la
 examples_udpong_SOURCES = examples/udpong.c examples/common.c
 examples_udpong_LDADD = $(top_builddir)/src/librdmacm.la
+examples_cmtime_SOURCES = examples/cmtime.c examples/common.c
+examples_cmtime_LDADD = $(top_builddir)/src/librdmacm.la
 
 librdmacmincludedir = $(includedir)/rdma
 infinibandincludedir = $(includedir)/infiniband
